op-program/client/l2/engineapi/test/l2_engine_api_tests.go

```diff
@@ -442,7 +442,9 @@ func (h *testHelper) getPayload(id *eth.PayloadID) *eth.ExecutionPayloadEnvelope
 
 func (h *testHelper) callNewPayload(envelope *eth.ExecutionPayloadEnvelope) (*eth.PayloadStatusV1, error) {
 	n := new(big.Int).SetUint64(uint64(envelope.ExecutionPayload.BlockNumber))
-	if h.backend.Config().IsCancun(n, uint64(envelope.ExecutionPayload.Timestamp)) {
+	if h.backend.Config().IsIsthmus(uint64(envelope.ExecutionPayload.Timestamp)) {
+		return h.engine.NewPayloadV4(h.ctx, envelope.ExecutionPayload, []common.Hash{}, envelope.ParentBeaconBlockRoot, nil)
```

Comment:
the requests shouldn't be nil here:
```suggestion
		return h.engine.NewPayloadV4(h.ctx, envelope.ExecutionPayload, []common.Hash{}, envelope.ParentBeaconBlockRoot, []hexutil.Bytes{})
```